### DISCUSSION TITLE
Stream only part of screen

### DISCUSSION THREAD

**Discussion by underdoeg** [LizardByte/community#375] [Sunshine] [answered]
I use linux (Gnome, X11) on my workstation and have a samsung S8 Tab with S Pen support. Now with touch and preliminary stylus support added to master https://github.com/LizardByte/Sunshine/pull/2017 sunshine can be used as a drawing tablet in blender or krita. Since my workstation has two displays (of which one is UHD) it would be nice to stream only part of a screen (or a specific window) in the tablets native resolution. (for example the drawing area of blender) Is this possible?

x11grab already does use offset and size parameter: https://github.com/LizardByte/Sunshine/blob/a10ec3a98ac6a3f5618dc53469f3c5b48dab36ee/src/platform/linux/x11grab.cpp#L535

but I could not figure out how to override them, the values seem to be automatically set to the screen area
https://github.com/LizardByte/Sunshine/blob/a10ec3a98ac6a3f5618dc53469f3c5b48dab36ee/src/platform/linux/x11grab.cpp#L448

**Comment by ReenigneArcher** [comment, marked as answer]
> Is this possible?

Nope, Sunshine streams your entire desktop only.
